docs/guide/slots.md

```diff
@@ -145,6 +145,51 @@ end
 <% end %>
 ```
 
+## Referencing slots
+
+You can access the instance of a slot in your ruby file by using the `before_render` [lifecycle method](./lifecycle.md).
```

Comment:
```suggestion
You can access the instance of a slot in your component class by using the `before_render` [lifecycle method](./lifecycle.md).
```

Reply:
Good suggestion, I always trip over this type of terminology :)